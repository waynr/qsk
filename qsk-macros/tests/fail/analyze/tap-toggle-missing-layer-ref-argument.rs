use qsk_macros::layer;

fn main() {
    layer!(
        ModLayer: {
            F -> TapToggle(),
        },
    );
}
