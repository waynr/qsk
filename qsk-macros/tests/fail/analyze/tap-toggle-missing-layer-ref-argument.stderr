error: missing layer ref argument
 --> tests/fail/analyze/tap-toggle-missing-layer-ref-argument.rs:6:18
  |
6 |             F -> TapToggle(),
  |                  ^^^^^^^^^
